test/extended/builds/pipeline.go

```diff
@@ -200,7 +201,7 @@ var _ = g.Describe("[builds][Slow] openshift pipeline build", func() {
 			br.AssertSuccess()
 
 			g.By("confirm all the log annotations are there")
-			_, err = jenkins.ProcessLogURLAnnotations(oc, br)
+			err = jenkins.ProcessLogURLAnnotations(oc, br, !useSnapshotImage)
```

Comment:
Cool on the deferring addressing when we deprecate v1 ... that said, the `useSnapshotImage` is in fact how we tell which image we are using (maybe I'm misinterpreting what you wrote?).  That variable being true means this is a PR test and hence, for the time being, is based on jenkins v1.  So `ProcessLogURLAnnotations` will only check the raw and chrome wrappred log urls if we are PR testing.  For our overnight extended test runs, we'll know via `useSnapshotImage` that it is our vanilla jenkins v2 image and we'll also check the blue ocean url.